components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -4408,6 +4409,94 @@ public int compare(Application o1, Application o2) {
         return applications;
     }
 
+    /**
+     *
```

Comment:
Add a proper Javadoc comment explaining the functionality.

Reply:
Javadoc comment is added